scripts/qa_crowbarsetup.sh

```diff
@@ -2504,7 +2504,7 @@ function custom_configuration
             fi
         ;;
         aodh)
-            if [[ $hacloud = 1 ]] ; then
+            if [[ $hacloud = 1 ]] && ! iscloudver 9plus; then
```

Comment:
`iscloudver 9minus` ? (Or `8minus`, I'm not sure if its `<` or `<=`)

Reply:
`! iscloudver 9plus` or `iscloudver 8minus` is equivalent and both correct.
But this would also match for cloud version 6 but aodh is only in cloud 7 and 8 (see other change).

So maybe change it here as well to something like this:
```
if [[ $hacloud = 1 ]] && ( iscloudver 7 || iscloudver 8 ) ; then
```